src/stores/UserStore.js

```diff
@@ -2,6 +2,9 @@ import {extendObservable} from 'mobx';
 import {browserHistory} from 'react-router';
 import React from 'react';
 
+// Please put some space between your class methods below
```

Comment:
Not sure what you mean. Looks like there's space between the methods. Did you want space between the binds?